hack/deploy-hub-local/sno-metal3.yml

```diff
@@ -0,0 +1,12 @@
+version: stable
+tag: 4.10
+sno: true
+sno_virtual: true
+network_type: OVNKubernetes
+cluster: test-ci
+domain: alklabs.com
+memory: 25000
```

Comment:
/hold WDYT about 32gb for the vm? I'm getting another test with insufficient memory for assisted-service

Reply:
increased to 40